lib/assignDisabledRanges.js

```diff
@@ -2,12 +2,12 @@
 "use strict"
 const _ = require("lodash")
 
-const COMMAND_PREFIX = "stylelint-"
-const disableCommand = COMMAND_PREFIX + "disable"
-const enableCommand = COMMAND_PREFIX + "enable"
-const disableLineCommand = COMMAND_PREFIX + "disable-line"
-const disableNextLineCommand = COMMAND_PREFIX + "disable-next-line"
-const ALL_RULES = "all"
+const COMMAND_PREFIX/*: string*/ = "stylelint-"
+const disableCommand/*: string*/ = COMMAND_PREFIX + "disable"
+const enableCommand/*: string*/ = COMMAND_PREFIX + "enable"
+const disableLineCommand/*: string*/ = COMMAND_PREFIX + "disable-line"
+const disableNextLineCommand/*: string*/ = COMMAND_PREFIX + "disable-next-line"
+const ALL_RULES/*: string*/ = "all"
```

Comment:
There is no need to annotate constants, since Flow can understand them just fine.